### PR TITLE
feat: migrate the changelog command

### PR DESCRIPTION
**Pull request by M0nkeySan**
add a changelog command to display the commits of the last 3 tags.

Also include a link to the ticket if a ticket_id is detected in the commit message.

A new env variable is required to configure the ticket management URL : `TICKET_MANAGEMENT_URL_PATTERN` 

resolve part of #16

### CODE REVIEW

File: src/changelog/buildChangelogModalView.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                text: 'Previous release tag',
+              },
+            },
+            {
+              type: 'context',
+              block_id: 'changelog-release-tag-info-block',
+              elements: [
+                {
+                  type: 'plain_text',
+                  text: 'This should be changed only whether the previous release has been aborted.',
```
Comment (greg0ire) [inline src/changelog/buildChangelogModalView.ts:125]:
```suggestion
                  text: 'This should be changed only if the previous release has been aborted.',
```

File: README.md [hunk 2]
```diff
@@ -213,6 +213,11 @@ Create a `.env` file containing the following variables:
 
   The gitlab URL of your organization
 
+- `TICKET_MANAGEMENT_URL_PATTERN`
+
+  The ticket management URL pattern for your organization. It must contain the `{ticketId}` matcher to be replaced by the ticket ID,
+  for instance `https://my-ticket-management.com/view/{ticketId}`.
```
Comment (greg0ire) [inline README.md:219]: Can you maybe described what feature this supports?

File: src/core/viewBuilders/buildHelpMessage.ts [hunk 3]
```diff
@@ -7,6 +7,7 @@ export function buildHelpMessage(channelId: string): ChatPostMessageArguments {
     text: `\
 Here are the available commands:
 
+- /homer changelog - Generate changelogs for any Gitlab project using release tags.
```
Comment (greg0ire) [inline src/core/viewBuilders/buildHelpMessage.ts:10]: What does that mean in practice? Is the changelog published somewhere?

Comment (M0nkeySan) [reply src/core/viewBuilders/buildHelpMessage.ts:10]: The changelog is displayed in a slack modal, nothing is published.

Do you want me to update the message to specify it ?

**Review by greg0ire — APPROVED** [review]

**Comment by cicoub13** [comment]
Can you add the command to [README](https://github.com/ManoManoTech/homer/blob/58192c95a4559616c95b2184e378dc730e7662af/README.md?plain=1#L35) ?

**Pull request merged**